src/ext.js

```diff
@@ -104,120 +105,9 @@ const getStyleSettings = (env) => {
-                  component: 'integer',
-                  // placeholder: () => parseInt(styleService.getStyle('header', 'fontSize'), 10),
-                  maxlength: 3,
-                  change(data) {
-                    data.header.fontSize = Math.max(5, Math.min(300, Math.floor(data.header.fontSize)));
-                  },
-                },
-                headerFontColor: {
-                  show: true,
-                  ref: 'header.fontColor',
-                  translation: 'ThemeStyleEditor.style.headerFontColor',
-                  type: 'object',
-                  component: 'color-picker',
-                  dualOutput: true,
-                },
-                fontSize: {
-                  show: true,
-                  translation: 'ThemeStyleEditor.style.cellFontSize',
-                  ref: 'content.fontSize',
-                  component: 'integer',
-                  // placeholder: () => parseInt(styleService.getStyle('content', 'fontSize'), 10),
-                  maxlength: 3,
-                  change(data) {
-                    data.content.fontSize = Math.max(5, Math.min(300, Math.floor(data.content.fontSize)));
-                  },
-                },
-                fontColor: {
-                  show: true,
-                  ref: 'content.fontColor',
-                  translation: 'ThemeStyleEditor.style.cellFontColor',
-                  type: 'object',
-                  component: 'color-picker',
-                  dualOutput: true,
-                },
-                hoverEffect: {
-                  show: true,
-                  ref: 'content.hoverEffect',
-                  translation: 'ThemeStyleEditor.style.hoverEffect',
-                  type: 'boolean',
-                  component: 'switch',
-                  options: [
-                    {
-                      value: true,
-                      translation: 'properties.on',
-                    },
-                    {
-                      value: false,
-                      translation: 'properties.off',
-                    },
-                  ],
-                },
-                hoverColor: {
-                  show: (data) => !!data.content.hoverEffect,
-                  ref: 'content.hoverColor',
-                  translation: 'ThemeStyleEditor.style.hoverStyle',
-                  type: 'object',
-                  component: 'color-picker',
-                  dualOutput: true,
-                },
-                hoverFontColor: {
-                  show: (data) => !!data.content.hoverEffect,
-                  ref: 'content.hoverFontColor',
-                  translation: 'ThemeStyleEditor.style.hoverFontStyle',
-                  type: 'object',
-                  component: 'color-picker',
-                  dualOutput: true,
-                },
-              },
-            },
-          },
-        },
+        env.flags.isEnabled('SENSECLIENT_PS_17478_STYLINGPANEL_STRAIGHT_TABLE_OSS')
```

Comment:
This is failing in the unit tests since isEnabled is false. Easiest would be to add optional chaining and the tests would run with `createStylingEditorDefinition`

Reply:
Added optional chaining